.vscode/settings.json

```diff
@@ -1,5 +1,7 @@
 {
```

Comment:
This file should never be in Github, add `.vscode/` to `.gitignore`, or better, you global `.gitignore`. https://vizzuality.slack.com/archives/C062HTHA5/p1684837965985219?thread_ts=1684837839.428609&cid=C062HTHA5